sound/soc/sof/pcm.c

```diff
@@ -670,6 +670,11 @@ static int sof_pcm_dai_link_fixup(struct snd_soc_pcm_runtime *rtd,
 		return -EINVAL;
 	}
 
+	if (!dai->dai_config) {
+		dev_dbg(sdev->dev, "no DAI config for %s!\n", rtd->dai_link->name);
+		return 0;
```

Comment:
Can you comment this assumption, I'm assuming this is for VFE only ? what impact does this have ?

Reply:
A comment added, I don't see any impact on the host side, what can this break?